package.json

```diff
@@ -38,17 +38,17 @@
     "clean": "lerna clean -y",
     "check:node": "check-node-version --node 16 --npm 8",
     "fix": "npm run fix:js && npm run fix:style",
-    "fix:js": "eslint --fix --ext .ts,.tsx,.js,.jsx src",
-    "fix:style": "stylelint --fix src/**/*.{scss,css}",
-    "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
-    "format:check": "prettier --list-different \"src/**/*.{js,jsx,ts,tsx,css,scss,html}\"",
+    "fix:js": "eslint --fix --ext .ts,.tsx,.js,.jsx src tests/performance-tests/src useful-scripts",
+    "fix:style": "stylelint --fix {src,tests/performance-tests/src,useful-scripts}/**/*.{scss,css}",
```

Comment:
We don't have any css or scss files inside these directories (and it wouldn't be too valuable imo) so I'd suggest getting rid of these extra paths

Reply:
Oh yeah good call :see_no_evil:

Reply:
Done ta